app/assets/stylesheets/app.scss

```diff
@@ -39,3 +39,7 @@
     width: 256px;
     height: 256px;
 }
+
+.badges {
```

Comment:
Class that does the same thing exists in bootstrap already, `.text-nowrap`

Reply:
I don't understand web development. AFAIU, CSS is meant for styling while HTML defines the document's structure. Therefore HTML should not reference anything like ".text-nowrap", but instead defer that to CSS, right?

Reply:
Yeah, bootstrap is a slightly complex set of css and js files :P